tests/cms/views/link_replace/test_link_actions.py

```diff
@@ -0,0 +1,408 @@
+) -> None:
+    """
+    Test whether link replacement in the network management per pencil icon in the column "options" work correctly.
+    """
+    listeners.tests_running = True
+
+    settings.LANGUAGE_CODE = "en"
+    client, role = login_role_user
+    region, before, after = parameter
+
+    url_object = Url.objects.filter(url=OLD_URL).first()
+
+    assert Link.objects.filter(url=url_object).count() == before
+    assert not Link.objects.filter(url__url=NEW_URL).exists()
+
+    kwargs = {
+        "url_filter": "valid",
+        "url_id": url_object.id,
+    }
+    if not region == "network_management":
+        kwargs.update(
+            {
+                "region_slug": region,
+            }
+        )
+
+    action_url = reverse(
+        "edit_url",
+        kwargs=kwargs,
+    )
+    with enable_listeners():
+        response = client.post(
+            action_url,
+            data={
+                "url": NEW_URL,
+            },
+        )
+
+    entitled_roles = (
+        STAFF_ROLES
+        if region == "network_management"
+        else STAFF_ROLES + [MANAGEMENT, EDITOR]
+    )
+
+    if role in entitled_roles:
+        assert response.status_code == 302
+
+        assert_message_in_log(
+            "SUCCESS  URL was successfully replaced",
+            caplog,
+        )
+
+        assert Link.objects.filter(url__url=OLD_URL).count() == after
+        assert Link.objects.filter(url__url=NEW_URL).count() == before - after
+
+    elif role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location")
+            == f"{settings.LOGIN_URL}?next={action_url}"
+        )
+        assert Link.objects.filter(url__url=OLD_URL).count() == before
+        assert not Link.objects.filter(url__url=NEW_URL).exists()
+    else:
+        assert response.status_code == 403
+        assert Link.objects.filter(url__url=OLD_URL).count() == before
+        assert not Link.objects.filter(url__url=NEW_URL).exists()
+
+    listeners.tests_running = False
+
+
+# ----------- Test for search&replace -----------#
+# string to be replaced
+SEARCH = "/augsburg/de/willkommen/"
+# string that appear after search&replace
+REPLACE = "/i/am/replaced/"
+# This URL is used once in region Nürnberg and three times in Augsburg
+# If bulk action is conducted in the network management, all of them must be affected,
+# while only three of them if done in the broken link list of Augsburg
+SEARCH_REPLACE_TARGET_URL = "https://integreat.app/augsburg/de/willkommen/"
+TARGET_URL_AFTER_REPLACE = "https://integreat.app/i/am/replaced/"
+# Parameters for test
+# (
+# <network management or region>,
+# <number of links with url=SEARCH_REPLACE_TARGET_URL before successful search&replace>,
+# <number of links with url=SEARCH_REPLACE_TARGET_URL which must stay unchanged after successful search&replace>
+# )
+search_replace_parameters = [("network_management", 4, 0), ("augsburg", 4, 1)]
+
+
+@pytest.mark.skip
```

Comment:
Why do we skip this one?

Or put differently, when is this supposed to be run? 😅 If I remove this line the test seems to run forever, but is there a situation where this test *is* run and is able to complete?